velox/exec/tests/utils/HiveConnectorTestBase.cpp

```diff
@@ -79,19 +87,41 @@ void HiveConnectorTestBase::writeToFile(
     const std::vector<RowVectorPtr>& vectors,
     std::shared_ptr<dwrf::Config> config,
     const TypePtr& schema) {
-  velox::dwrf::WriterOptions options;
-  options.config = config;
-  options.schema = schema;
   auto localWriteFile = std::make_unique<LocalWriteFile>(filePath, true, false);
   auto sink = std::make_unique<dwio::common::WriteFileSink>(
       std::move(localWriteFile), filePath);
   auto childPool = rootPool_->addAggregateChild("HiveConnectorTestBase.Writer");
-  options.memoryPool = childPool.get();
-  facebook::velox::dwrf::Writer writer{std::move(sink), options};
-  for (size_t i = 0; i < vectors.size(); ++i) {
-    writer.write(vectors[i]);
+  switch (fileFormat_) {
+#ifdef VELOX_ENABLE_PARQUET
+    case dwio::common::FileFormat::PARQUET: {
+      facebook::velox::parquet::WriterOptions writerOptions;
```

Comment:
This issue is actually about refactoring and unifying Writer interfaces. See how we made the dwrf::Writer and parquet::Writer classes to all inherit dwio::common::Writer. We need to do similar to the WriterOptions.

Reply:
Edited WriterOptions for DWRF and Parquet to inherit from dwio::common::WriterOptions.

```
struct WriterOptions : dwio::common::WriterOptions
{
...
}
```